dxvk.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+
+	"github.com/vinegarhq/vinegar/util"
+)
+
+// this file, created on DXVK installation and removed at DXVK uninstallation,
+// is an easy way to tell if DXVK is installed, necessary for automatic installation
+// and uninstallation of DXVK.
+var DxvkInstallMarker = filepath.Join(Dirs.Prefix, ".vinegar-dxvk")
+
+const DXVKVER = "2.1"
+
+func DxvkStrap() {
+	_, err := os.Stat(DxvkInstallMarker)
+
+	// Uninstall DXVK when DXVK is disabled, and if the file exists (no error)
+	if err == nil {
+		if !Config.Dxvk {
+			DxvkUninstall()
+		}
+
+		return
+	}
+
+	DxvkInstall()
+}
+
+func DxvkInstall() {
+	log.Printf("Installing DXVK %s", DXVKVER)
+
+	tarName := "dxvk-" + DXVKVER + ".tar.gz"
+	tarPath := filepath.Join(Dirs.Cache, tarName)
+	url := "https://github.com/doitsujin/dxvk/releases/download/v" + DXVKVER + "/" + tarName
```

Comment:
If we are using md5 for Roblox, we might as well begin storing md5 sums for DXVK as well.